src/services/utilities.ts

```diff
@@ -1337,11 +1337,6 @@ namespace ts {
     export function getPropertySymbolFromBindingElement(checker: TypeChecker, bindingElement: ObjectBindingElementWithoutPropertyName) {
         const typeOfPattern = checker.getTypeAtLocation(bindingElement.parent);
         const propSymbol = typeOfPattern && checker.getPropertyOfType(typeOfPattern, bindingElement.name.text);
-        if (propSymbol && propSymbol.flags & SymbolFlags.Accessor) {

```

Comment:
This is also used in `updateImportsInOtherFiles` is there test for this?

Reply:
Fixed